pytorch_lightning/trainer/connectors/checkpoint_connector.py

```diff
@@ -375,6 +434,16 @@ def dump_checkpoint(self, weights_only: bool = False) -> dict:
 
         return checkpoint
 
+    def get_loops_state_dict(self):
+        return {"result_collections": self.get_result_collections_state_dict()}
+
+    def get_result_collections_state_dict(self):
+        return {
+            RunningStage.TRAINING.value: self.trainer.train_loop.results.state_dict(),
```

Comment:
```suggestion
            RunningStage.TRAINING.value: self.trainer.fit_loop.results.state_dict(),
```
recently changed name

Reply:
Done !